scrapy/downloadermiddlewares/httpauth.py

```diff
@@ -15,17 +16,28 @@ class HttpAuthMiddleware:
 
     @classmethod
     def from_crawler(cls, crawler):
-        o = cls()
+        o = cls()   
         crawler.signals.connect(o.spider_opened, signal=signals.spider_opened)
         return o
 
     def spider_opened(self, spider):
-        usr = getattr(spider, 'http_user', '')
-        pwd = getattr(spider, 'http_pass', '')
-        if usr or pwd:
-            self.auth = basic_auth_header(usr, pwd)
+        pass
```

Comment:
No need for this method to be here if it does nothing.

Reply:
Okay I will remove it then.